src/test/kotlin/org/jitsi/jibri/util/RefreshingPropertyTest.kt

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright @ 2018 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi.jibri.util
+
+import io.kotest.core.spec.style.ShouldSpec
+import io.kotest.matchers.shouldBe
+import io.mockk.spyk
+import org.jitsi.jibri.helpers.FakeClock
+import org.jitsi.jibri.helpers.minutes
+import org.jitsi.jibri.helpers.seconds
+import java.time.Duration
+
+class RefreshingPropertyTest : ShouldSpec({
+    val clock: FakeClock = spyk()
```

Comment:
`FakeClock` shouldn't need mocking if you take the version we have in jmt.

Reply:
The one in JMT doesn't help me much here, unfortunately.  We'll have to move that to `jicoco-test` at some point.

Reply:
Oh, maybe you meant copy the one from JMT into here.  This is working fine for now, I'll just wait until we get that one in a common place.